src/BottomSheetView.tsx

```diff
@@ -132,51 +132,50 @@ export function BottomSheetView({ state, descriptors }: Props) {
   return (
     <>
       {firstScreen.render()}
-      <Overlay>
```

Comment:
Overlay is needed on iOS to make sure that the modal is presented on top. Without it, modals would show behind screens sometimes (e.g. modals would show behind screens presented modally)